pilot/pkg/bootstrap/server.go

```diff
@@ -291,6 +300,38 @@ func (s *Server) initMonitor(args *PilotArgs) error {
 	return nil
 }
 
+func (s *Server) initMeshConfigProtocolClient(args *PilotArgs) error {
+	serverAddr := args.MCPServerAddr
+	conn, err := grpc.Dial(serverAddr, grpc.WithInsecure())
```

Comment:
FYI: I am working on the mTLS security model for the MCP protocol. Over time, the WithInsecure() option should be replaced with the right cert loading code.
But for the initial checkin, this should be fine.

Reply:
Yes - it MUST have mtls, but ok to start without.